server.go

```diff
@@ -51,6 +51,14 @@ type Server struct {
 }
 
 func NewServer(timeout time.Duration) *Server {
+	// The default is 65KB, however Kafka messages might typically be
+	// much larger than this, so we bump the limit to 512MB.
+	//
+	// Keep in mind that the Redis protocol also specifies strings may be
+	// up to 512MB, although there are plans to make them even bigger
+	// (see https://github.com/antirez/redis/issues/757).
+	redisproto.MaxBulkSize = 1024 * 512
```

Comment:
This is 512 **k**B I guess :)

Reply:
Should we have Redis as a direction of max message size or think of the messages we pass along to Kafka so we do not propagate errors? Following Kafka's `message.max.bytes` seems more reasonable.

Either way, we should probably document our "protocol" max size in the README.

Reply:
Bumped to 32MB.